src/ducks/auth.duck.js

```diff
@@ -5,6 +5,7 @@ import { storableError } from '../util/errors';
 import * as log from '../util/log';
 
 const authenticated = authInfo => authInfo && authInfo.isAnonymous === false;
+const loggedInAs = authInfo => authInfo && authInfo.isLoggedInAs === true;
```

Comment:
These can be updated to modern JS:

```suggestion
const loggedInAs = authInfo => authInfo?.isLoggedInAs === true;
```